src/pages/about-page/about-page.module.css

```diff
@@ -0,0 +1,104 @@
+.about {
+  padding: 40px 0 20px;
+}
+
+.about__inner {
+  display: flex;
+  justify-content: space-between;
+  align-items: start;
+  gap: 80px;
+}
+
+.about__inner-text {
+  display: flex;
+  flex-direction: column;
+  max-width: 45%;
+  padding-bottom: 70px;
+}
+
+.about__inner a {
+  color: #111;
+  font-weight: 400;
+  text-decoration: underline;
+}
+
+.about__inner p {
```

Comment:
Please correct `line-height` according to design

Reply:
And please add some decent class names for the paragraphs. Try to avoid styling by tags